budgetApp1/budgetApp1/TransactionsRepository.swift

```diff
@@ -1,42 +1,21 @@
 //
-//  BudgetRepository.swift
+//  TrabnsactionsRepository.swift
 //  budgetApp1
 //
-//  Created by Lehlohonolo Nkadimeng on 2024/03/26.
+//  Created by Lehlohonolo Nkadimeng on 2024/05/15.
 //
 
 import Foundation
 
-typealias BudgetsResult = (Result<BudgetData, APIError>) -> Void
-typealias AccountResult = (Result<AccountData, APIError>) -> Void
 typealias TransactionResult = (Result<TransactionData, APIError>) -> Void
 
-protocol BudgetRepositoryType: AnyObject {
-    func getBudgets(completion: @escaping(BudgetsResult))
-}
-
-protocol AccountRepositoryType: AnyObject {
-    func getAccounts(completion: @escaping(AccountResult))
-}
-
+// MARK: - Repository Type
 protocol TransactionRepositoryType: AnyObject {
     func getTransactions(completion: @escaping(TransactionResult))
```

Comment:
fetchTransactions

Reply:
refactored for all repositories and viewmodels.